Add Android TV 0.12 post

nielsvanvelzen opened the pull request:
Add a blog post for the Android TV 0.12 release. Should be merged around the same time as the apps get published in GPlay/Amazon Appstore. (assigning @anthonylavado for that task 😉).

I'd like to get some feedback on the writing, as English is not my primary language and writing is definitely not one of my skills :p

I need to retake the screenshots (accidently used the debug logo.. whoops) so ignore those for now.

<details>
<summary>Rendered post (First commit)</summary>

![image](https://user-images.githubusercontent.com/2305178/134800756-4976879d-b332-4465-a0e2-6207ce21849a.png)

</details>

<details>
<summary>Rendered post ((almost) Latest commit)</summary>

![image](https://user-images.githubusercontent.com/2305178/135143724-9be74123-19ed-42f6-9a21-46eeeb2de85a.png)

</details>

nielsvanvelzen commented on content/posts/android-tv-12.md:
```diff
@@ -0,0 +1,143 @@
+---
+title: Android TV v0.12
+subtitle: This is the subtitle
```
You forgot to add a subtitle, idiot

MrChip53 replied:
https://youtu.be/jRn48HxssPI?t=15s

thornbill commented on content/posts/android-tv-12.md:
```diff
@@ -0,0 +1,143 @@
+
+- [@linetrimmer](https://github.com/linetrimmer)
+- [@Froghut](https://github.com/Froghut)
+- [@jellyfin-bot](https://github.com/jellyfin-bot)
```
Should we include the bot as a contributor? 😄

nielsvanvelzen replied:
Nope, I did remove dependabot and github-actions[bot] but forgot to remove the jellyfin-bot.

MrChip53 commented:
I'll check this out fully when I get home tonight!

nielsvanvelzen commented:
Updated the blog post with fixed screenshots (release logo) and updated the "Themes" section (it mentioned the screenshots using Purple Haze but they don't). Post should be ready now :)

Maxr1998 commented on content/posts/android-tv-12.md:
```diff
@@ -0,0 +1,142 @@
+
+## New authentication flow
+
+We got a lot of complaints about our authentication flow. It wasn't obvious how to enable the auto log in option and managing multiple servers or users was not an easy task. We also didn't like this part of the app, and decided to completely revamp it! The rewritten sign in screen looks more modern and is much easier to use. There is a new help section that links to our documentation to help new  ...
```
```suggestion
We got a lot of complaints about our authentication flow. It wasn't obvious how to enable the auto-login option and managing multiple servers or users was not an easy task. We also didn't like this part of the app, and decided to completely revamp it! The rewritten sign-in screen looks more modern and is much easier to use. There is a new help section that links to our documentation to help new users get started with Jellyfin. The auto-discovery feature of the app now shows all servers instead of the first one, you can select one of those or manually enter your server address to connect . You can then proceed to add a user and start using the app. Users are automatically saved now with auto-login enabled by default.
```

Maxr1998 commented on content/posts/android-tv-12.md:
```diff
@@ -0,0 +1,142 @@
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-settings.png" />
+</div>
+
+The user interfacer when browsing inside libraries got some slight changes too. We're hoping to completely revamp this part of the app at some point, but that doesn't stop us from making it slightly better right now.
```
Or "didn't" instead of won't if the changes are already in the update.

```suggestion
The user interface when browsing inside libraries got some slight changes too. We're hoping to completely revamp this part of the app at some point, but that won't stop us from making it slightly better now.
```

Maxr1998 commented on content/posts/android-tv-12.md:
```diff
@@ -0,0 +1,142 @@
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-home.png" />
+</div>
+
+The settings screen got a refresh too. It allows you to more easily change settings. Some new settings got added to customize the app to your own tastes.
```
```suggestion
The settings screen got a refresh too. It allows you to more easily change settings. Some new settings got added to customize the app to your own taste.
```

Maxr1998 commented on content/posts/android-tv-12.md:
```diff
@@ -0,0 +1,142 @@
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-player.png" />
+</div>
+
+The new video player design is just the beginning. A project to rewrite the playback code behind it is already in the works. This rewrite should help with the crashes and unnecessary transcoding that happens sometimes. It will also give us the opportunity to add new features like SyncPlay. But this doesn't mean the current code isn't being worked on anymore! We did fix some issues with the curren ...
```
```suggestion
The new video player design is just the beginning. A project to rewrite the playback code behind it is already in the works. This rewrite should help with the crashes and unnecessary transcoding that happens sometimes. It will also give us the opportunity to add new features like SyncPlay. But this doesn't mean the current code isn't being worked on anymore! We did fix some issues with the current video player code. Notable are changes for Fire TV to direct play more often and a lot of crashes got fixed.
```

Maxr1998 commented on content/posts/android-tv-12.md:
```diff
@@ -0,0 +1,142 @@
+Fortunately we've worked hard to make sure we did feel confident and here we are! Starting from this release we're changing some things
+to make sure the next release won't take this long. Our brand new SDK that is already used in our Android app and third party apps is one of the tools helping us with faster releases. By having a more type-safe base we can prevent tons of crashes, allowing us to focus on actual features and bugs.
+
+We're already working on some new stuff like the earlier mentioned rewrite of the playback code. We're using feature flags for this so we can work on this code while still being able to publish new versions. Other changes that we're looking into are more improvements to the user interface, better Live TV, better music support and more bugfixing!
```
```suggestion
We're already working on some new stuff like the earlier mentioned rewrite of the playback code. We're using feature flags for this so we can work on this code while still being able to publish new versions. Other changes that we're looking into are more improvements to the user interface, better Live TV, better music support and more bugfixes!
```

Maxr1998 reviewed (changes requested):
Some slight tweaks to wording and grammar (and one typo).
